src/server.ts

```diff
@@ -170,10 +170,8 @@ export async function startServer (port: number, hardware: Hardware = 'v3', com:
     async executeMotion(motion: Motion, _progress: [number, number]): Promise<void> {
       await ebb.executeMotion(motion);
     },
-    async postCancel(): Promise<void> {
-      const device = Device(ebb.hardware);
```

Comment:
@drskullster did removing this line improve anything?  Were there any side effects?

(I don't know its original purpose)

Reply:
@alexrudd2 I removed it because the Device object was used to set the pen to a defined position (`device.penPctToPos(50)`). We can use the `ebb` object for that, so we don't need the `Device` object.